pkg/broker/cluster_params.go

```diff
@@ -0,0 +1,51 @@
+// kibosh
+//
+// Copyright (C) 2015-Present Pivotal Software, Inc. All rights reserved.
+
+// This program and the accompanying materials are made available under
+// the terms of the under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package broker
+
+import (
+	"encoding/json"
+
+	"github.com/cf-platform-eng/kibosh/pkg/config"
+)
+
+type configParams struct {
+	ClusterConfig clusterConfig `json:"clusterConfig"`
+}
+
```

Comment:
What is this type doing that `config.ClusterCredentials` can't? Don't see why you've added a second type with the same data (server, token, ca).

Reply:
Mainly so this can be expanded in the future.  Tomorrow configParams might have more fields (like a cluster name) and this should make it easier to keep compatibility.